cime_config/cesm/machines/Makefile

```diff
@@ -776,8 +770,8 @@ cleanrof:
 	cd $(EXEROOT)/rof/obj ; $(RM) -f *.o *.mod
 
 cleanlnd:
-	$(RM) -f $(LIBROOT)/liblnd.a
-	cd $(EXEROOT)/lnd/obj ; $(RM) -f *.o *.mod
+	$(RM) -f $(LNDLIBDIR)/$(LNDLIB)
```

Comment:
I'm running the yellowstone-intel aux_clm45 test suite on this: I realized it was more efficient for me to just do this myself. I'll add a note later (probably this evening) with the results.